### PR TITLE
Compat shadow RTL support

### PR DESCRIPTION
**Pull request by goncalossilva**

### CODE REVIEW

File: MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDelegate.java [hunk 1]
```diff
@@ -68,9 +71,14 @@ public CompatElevationDelegate(View view, AttributeSet attrs, int defStyleAttr)
             mElevation = a.getDimensionPixelOffset(R.styleable.CompatElevationDelegate_elevation, 0);
             mCornerRadius = a.getDimensionPixelOffset(R.styleable.CompatElevationDelegate_cornerRadius, 0);
             int shownShadows = a.getInt(R.styleable.CompatElevationDelegate_shownShadows, SHOW_ALL_SHADOWS);
-            mShowShadowLeft = (shownShadows & SHOW_SHADOW_LEFT) == SHOW_SHADOW_LEFT;
+            boolean isLtr = ViewCompat.getLayoutDirection(view) == ViewCompat.LAYOUT_DIRECTION_LTR;
+            boolean showShadowStart = (shownShadows & SHOW_SHADOW_START) == SHOW_SHADOW_START;
+            boolean showShadowEnd = (shownShadows & SHOW_SHADOW_END) == SHOW_SHADOW_END;
+            mShowShadowLeft = (shownShadows & SHOW_SHADOW_LEFT) == SHOW_SHADOW_LEFT
+                    || isLtr && showShadowStart || !isLtr && showShadowEnd
```
Comment (ab-helly) [inline MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDelegate.java]: Missed semicolon 😊

**Review by ab-helly — APPROVED** [review]
Thanks! Looks good, only left one comment 😄

File: MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDelegate.java (hunk 1 above)
Comment (goncalossilva) [reply MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDelegate.java]: Kotlin much 😅

File: MaterialLibrary/src/main/java/io/doist/material/drawable/WrapperDrawable.java [hunk 2]
```diff
@@ -142,6 +143,11 @@ protected void onBoundsChange(Rect bounds) {
     }
 
     @Override
+    public void setBounds(@NonNull Rect bounds) {
```
Comment (rastislavvasko) [inline MaterialLibrary/src/main/java/io/doist/material/drawable/WrapperDrawable.java]: Hmm, do we need to override this? 🤔 We set the wrapped drawable's bounds in `onBoundsChange`, which is called by `Drawable.setBounds`. Now that method won't be called at all and we don't actually set this drawable's bounds either.

Comment (ab-helly) [reply MaterialLibrary/src/main/java/io/doist/material/drawable/WrapperDrawable.java]: Nope, it was a really bad fix for an issue 😓 Found a proper one and will remove this commit.

**Pull request merged**